scripts/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.7)
+cmake_minimum_required(VERSION 3.6)
```

Comment:
Do we need an if(MSYS) 3.6 else 3.7 here?

Reply:
I don't think so. Kconfig is really easy to build. We don't need to restrict users to a recent CMake version for them to be able to build it. They can probably build it with whatever version they have installed.